### ISSUE TITLE
Does it supports for 64 bits DLL now?

### ISSUE THREAD

**Issue by areuu** [taviso/loadlibrary#91]

**Comment by taviso** [comment]
No, 32bit only - It would require a complete rewrite to support 64bit code.

**Comment by cube0x8** [comment]
Now yes, you find it here: https://github.com/cube0x8/loadlibrary/tree/x64

It's experimental, so if you try it and find some bugs, please report it :)
